webpack_loader/__init__.py

```diff
@@ -1,4 +1,4 @@
 __author__ = 'Owais Lone'
-__version__ = '0.8.3'
+__version__ = '0.8.4'
```

Comment:
I think version should be bumped in a separate release commit.

Reply:
Seems like a good idea, I'm just not yet sure what the best way to do this is.